tests/config.ini

```diff
@@ -0,0 +1,9 @@
+[Login]
```

Comment:
missing std header and some comments what this is for!
should it be in a sso subsection of the tests?
some additional info would be helpful.